bin/varnishstat/varnishstat.c

```diff
@@ -113,15 +113,13 @@ do_json_cb(void *priv, const struct VSC_point * const pt)
 	else
 		printf(",\n");
 
-	printf("  \"");
-	/* build the JSON key name.  */
-	printf("%s\": {\n", pt->name);
-	printf("    \"description\": \"%s\",\n", pt->sdesc);
+	printf("    \"%s\": {\n", pt->name);
+	printf("      \"description\": \"%s\",\n", pt->sdesc);
 
-	printf("    \"flag\": \"%c\", ", pt->semantics);
+	printf("      \"flag\": \"%c\", ", pt->semantics);
 	printf("\"format\": \"%c\",\n", pt->format);
-	printf("    \"value\": %ju", (uintmax_t)val);
-	printf("\n  }");
+	printf("      \"value\": %ju", (uintmax_t)val);
+	printf("\n    }");
```

Comment:
We should probably change this sequence of `printf` statements into a single one like you did line 137.

Reply:
not as easy to do since there are arguments to format?

Reply:
@Dridi can we consider this feedback addressed?

Reply:
I will do that myself once merged.